### PR TITLE
Increase test coverage

### PR DESCRIPTION
**Pull request by SerayaEryn**
As titled.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -372,6 +399,117 @@ test('Should compress json data (gzip) - global', t => {
...
+})
+
+test('Should not compress if content-type is a invalid type', t => {
+  t.plan(2)
+  const fastify = Fastify()
+  fastify.register(compressPlugin, { threshold: 0 })
+  const json = { hello: 'world' }
+
+  fastify.get('/', (req, reply) => {
+    reply.header('content-type', 'something/invalid')
```
Comment (delvedor) [inline test.js]: This test is not needed, `reply.type` is an alias of `reply.header('content-type', ` :)

File: test.js [hunk 2]
```diff
@@ -372,6 +399,117 @@ test('Should compress json data (gzip) - global', t => {
...
+
+  fastify.inject({
+    url: '/',
+    method: 'GET',
+    headers: {
+      'accept-encoding': 'deflate'
+    }
+  }, res => {
+    t.strictEqual(res.statusCode, 200)
+    t.notOk(res.headers['content-encoding'])
```
Comment (delvedor) [inline test.js]: Can you check also the payload?

**Comment by SerayaEryn** [comment]
I'll fix that as soon as possible.

**Comment by SerayaEryn** [comment]
I fixed the tests and I am now checking the payload in the tests.

**Pull request merged**